java/ql/lib/ext/java.lang.model.yml

```diff
@@ -114,6 +114,7 @@ extensions:
       - ["java.lang", "String", False, "indent", "", "", "Argument[this]", "ReturnValue", "taint", "manual"]
       - ["java.lang", "String", False, "intern", "", "", "Argument[this]", "ReturnValue", "taint", "manual"]
       - ["java.lang", "String", False, "join", "", "", "Argument[0..1]", "ReturnValue", "taint", "manual"]
+      - ["java.lang", "String", False, "lines", "()", "", "Argument[this]", "ReturnValue.Element", "taint", "df-generated"]
```

Comment:
Should this be `df-manual` instead, since we've reviewed it?

Reply:
Doh, yes. I see another 5 together as well.